tests/util/helpers.sh

```diff
@@ -105,22 +105,24 @@ _wait_for_statefulset() {
     fi
 }
 
-# Wait for Istio config to propagate
+# Wait for istio to be created
 # usage: _wait_for_istio <kind> <namespace> <name>
 _wait_for_istio() {
-    #local kind="$1"
-    #local namespace="$2"
-    #local name="$3"
-    local start=$(date +%s)
-    sleep 1s
-    # @TODO: Rewrite this to *NOT* use istioctl experimental wait, since it was removed
-    # https://github.com/istio/istio.io/issues/16429
-    #if ! istioctl experimental wait --for=distribution --timeout=10s "$kind" "$name.$namespace"; then
-        #echo "Failed distribution of $kind $name in namespace $namespace"
-        #istioctl ps
-        #echo "TEST: wait for failed, but continuing."
-    #fi
-    echo "Duration: $(($(date +%s)-start)) seconds"
+    local kind="$1"
+    local namespace="$2"
+    local name="$3"
+    local start_time
+    start_time="$(date +%s)"
+
+    if ! kubectl wait --for=create -n "$namespace" "$kind/$name" --timeout 30s; then
+        echo "Timed out waiting for $kind $name in namespace $namespace to be created."
+        kubectl get "$kind" -n "$namespace"
```

Comment:
not that useful to know what else is there, we're testing that something is _not_ there